docs/troubleshooting.md

```diff
@@ -0,0 +1,45 @@
+# Troubleshooting Guide
```

Comment:
Non-blocker: We have one in Confluence also. Happy to revisit final location.

For now, can you link this from `docs/README.md`

Reply:
I was going to mention: https://officefornationalstatistics.atlassian.net/wiki/spaces/NWP/pages/59650445/Troubleshooting+Guide